lightly/__init__.py

```diff
@@ -79,6 +79,11 @@
 
 
 import os
+from typing import Optional
+
+import packaging.version
```

Comment:
Didn't want to import `from packaging import version` because it could be confusing to have a `ligthly.version` attribute.